client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -92,7 +92,10 @@ class BusinessDetails extends Component {
 				extensionInstallationOptions[
 					'woocommerce-services:shipping'
 				] || extensionInstallationOptions[ 'woocommerce-services:tax' ],
+			install_google_listings_and_ads:
+				extensionInstallationOptions[ 'google-listings-and-ads' ],
 			install_jetpack: extensionInstallationOptions.jetpack,
+			install_mailpoet: extensionInstallationOptions.mailpoet,
 			install_wcpay:
 				extensionInstallationOptions[ 'woocommerce-payments' ],
 		} );
```

Comment:
Just noting here that it would be great to refactor this, so we don't have to specify the specific plugin names in the track. As this would still require us to rely on wc admin release sequence if we wanted to add a new plugin, when every thing else relies on the free features config.
Given this might get cherry picked we shouldn't do this as part of this PR, but maybe we should create a separate issue for this? @octaedro